lib/common/nuxt.config.js

```diff
@@ -207,9 +207,7 @@ export default {
     static: {
       prefix: true
     },
-    gzip: {
-      threshold: 0
-    },
+    compressor: undefined,
```

Comment:
How about setting gzip as deault value?

Reply:
You mean `compressor: { threshold: 0 }`?

Reply:
No bad idea!